contracts/SiphonReceiver.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { ERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/ERC20.sol";
+import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
+
+/**
+ * @title SiphonReciever
+ * @dev Takes rAURA deposits from rAURA on L1 and distributes them
+ *      When rewardClaimed is called on the Booster
+ */
+contract SiphonReceiver {
+    function queueRAura(uint256) external {
+        // TODO:
+        // Only callable from the L1 (via lzEndpoint)
+        // Mint rAURA to address(this)
+        //
+    }
+
+    function mint(address, uint256) external {
+        // TODO: transfer rAura to caller
+        // Only callable by the Boosters rewardClaimed
+    }
+
+    function queueNewRewards(uint256) external {
+        // TODO:
```

Comment:
Yep sounds good. Just transfer all the fees here. Only callable from booster. Can maybe have a seperate method called "flush" that sends to Lz and calls siphon, return etc that someone can call on a daily basis